src/components/Filter/TagList.vue

```diff
@@ -10,45 +10,50 @@
 
 <template>
   <div class="tags">
-    <ul
-      @keydown.left.capture.prevent="focusPrev"
-      @keydown.right.capture.prevent="focusNext"
-      @keydown.up.capture.prevent="focusPrev"
-      @keydown.down.capture.prevent="focusNext"
-      @keydown.delete.prevent.self="$emit('reset-filters')"
-      @keydown.meta.a.capture.prevent="$emit('select-all')"
-      @keydown.ctrl.a.capture.prevent="$emit('select-all')"
-      @keydown.exact.capture="handleKeydown"
-      @keydown.shift.exact.capture="handleKeydown"
-      @scroll="handleScroll"
-      :aria-label="ariaLabel"
+    <div
```

Comment:
This is a fix for the scrollbar, when trying to scroll vertically. It just needs an extra wrapper without padding.